libsql-server/src/lib.rs

```diff
@@ -632,62 +641,74 @@ where
         Arc<dyn Fn() -> InnerWalManager + Sync + Send + 'static>,
         Pin<Box<dyn Future<Output = anyhow::Result<()>> + Send + Sync + 'static>>,
     )> {
-        let wal_path = self.path.join("wals");
+        if self.use_custom_wal.is_none() {
+            tracing::info!("using sqlite3 wal");
+            return Ok((
+                Arc::new(|| Either3::A(Sqlite3WalManager::default())),
+                Box::pin(ready(Ok(()))),
+            ));
+        };
+
+        let use_custom_wal = self.use_custom_wal.unwrap();
```

Comment:
turn that into a match, it's more idiomatic

Reply:
yes, I have refactored this method now